src/index.html

```diff
@@ -12,11 +12,45 @@
     <title>Aurator</title>
 </head>
 <body class="app">
-    <img src="img/desktop/naprawa__foto.png" alt="foto">
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-12">
-                <h2>Aurator</h2>
+    <div class="container-fluid">
+        <div class="top">
+            <img class="" src="../img/desktop/banner__bg.png">
+            <div class="nav">
+                <div class="nav-logo">
+                    <img class="d-none d-sm-block" src="../img/desktop/nav__logo.png">
+                    <img class="d-block d-sm-none" src="../img/mobile/nav__logo.png">
+                </div>
+                <div class="nav-menu">
+                    <ul class="nav-menu__list main-menu">
+                        <li class="nav-menu__list__item"><a class="nav-link" href="#">NAPRAWA I REGENERACJA</a></li>
+                        <li class="nav-menu__list__item"><a class="nav-link" href="#">PRODUKCJA</a></li>
+                        <li class="nav-menu__list__item"><a class="nav-link" href="#">CENTRUM NAPRAW AWARYJNYCH LOKOMOTYW</a></li>
+                        <li class="nav-menu__list__item"><a class="nav-link" href="#">O FIRMIE</a></li>
+                        <li class="nav-menu__list__item"><a class="nav-link" href="#">KONTAKT</a></li>
+                    </ul>
+                    <ul class="nav-menu__list menu-kontakt">
+                        <li class="nav-menu__list__item"><a class="nav-link color-orange" href="#">+32 301 20 97</a></li>
```

Comment:
In case of `nav-link color-orange` I would rather use `nav-link nav-link--orange` (http://getbem.com/naming/).